src-backbone/tests/auth/auth_controller_test.js

```diff
@@ -0,0 +1,113 @@
+var CreateSuite = require('utils/create_suite');
+var AuthController = require('routers/auth/auth_controller');
+var SessionModel = require('models/session');
+var RootView = require('views/root_view');
+
+
+CreateSuite("Auth Controller", function() {
+    var app;
+    var auth_controller;
+
+    var root_view_stub;
+    var auth_layout_spy;
+    var auth_layout_stub;
+    var signup_view_stub;
+    var login_view_stub;
+
+    var helpers_mock;
+
+    beforeEach(function() {
+        app = {
+            session: new SessionModel(),
+            root_view: new RootView(),
+        };
+
+        root_view_stub = sinon.stub(app.root_view, 'showChildView');
+        auth_layout_spy = sinon.spy();
+        auth_layout_stub = sinon.stub().returns({ showChildView: auth_layout_spy });
+        signup_view_stub = sinon.stub();
+        login_view_stub = sinon.stub();
+
+        auth_controller = new AuthController({
+            app: app,
+            AuthLayout: auth_layout_stub,
+            SignupView: signup_view_stub,
+            LoginView: login_view_stub,
+        });
+
+        helpers_mock = sinon.mock(auth_controller.Helpers);
+    });
+
+    afterEach(function() {
+        helpers_mock.verify();
+    });
+
+    describe("signup()", function() {
```

Comment:
Nit but describes for functions should have a hash at the front, i.e. `"#signup()`